[BUG] @inngest/test@0.1.4 - module not found

**Describe the bug**

When upgrading to @inngest/test, the package seems unusable by TS.
The changelog says 

> Altered exports to now be namespaced by ./dist/; if you have directly imported files from @inngest/test, you may need to change the imports

I'm not sure what imports change should be, but with a minor version bump and no documentation change, it's probably not expected to break.

**To Reproduce**
Steps to reproduce the behavior:

1. `npm install @inngest/test@0.1.4`
2. In a jest TS file `import { InngestTestEngine } from '@inngest/test';`

Error: 

> Cannot find module '@inngest/test' or its corresponding type declarations.ts(2307)

**Expected behavior**

Being able to import the test engine.

--- a/.changeset/purple-insects-do.md
+++ b/.changeset/purple-insects-do.md
@@ -1,0 +1,5 @@
+---
+"@inngest/test": patch
+---
+
+Fix `@inngest/test` not shipping `dist/` files

--- a/packages/test/package.json
+++ b/packages/test/package.json
@@ -24,6 +24,9 @@
       "types": "./dist/index.d.ts"
     }
   },
+  "files": [
+    "dist"
+  ],
   "keywords": [
     "inngest",
     "test",

